circuits/core/manager.py

```diff
@@ -126,6 +127,41 @@ def __init__(self, timeout):
         self.tick_handler = None
 
 
+class EventQueue(object):
+    def __init__(self):
+        self._queue = deque()
+        self._priority_queue = []
+        self._counter = count()
+        self._flush_batch = 0
+
+    def __len__(self):
+        return len(self._queue) + len(self._priority_queue)
+
+    def drainFrom(self, other_queue):
```

Comment:
Do we want to keep the camelCase thing for new implementations? PEP8 wants lower_underscoe.